appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  matrix:
+  - julia_version: 0.7
+  - julia_version: 1
```

Comment:
`julia_version: 1` means the latest release version, i.e., `1.1`

Reply:
That's true. I'm torn about whether to test everything (which slows the tests down), or if only a subset, which versions to test? On one hand, version 1.0 is sometimes billed as the "safest" 1.x version, OTOH I suspect most julia users use the latest stable release.

Reply:
My idea is to test three versions:

* LTS versions : `1.0.3` (and more in the future)
* latest release: `1.1.0`
* nightly build, which allows failure

Reply:
There's a lot of sense in that list. At the current moment it is close to being comprehensive, though I guess it will be more selective when `1.2.0` becomes the latest stable release.

I also wonder about 0.7. I never use it anymore myself, but I wonder about what are probably a long list of projects that have not been updated yet. (I just updated another one of my lab's rarely-used repositories yesterday.) Whenever we drop testing on 0.7, I would say that it's important to also set julia 1.0 as the minimum supported version---otherwise we are claiming to support something that few of us are testing.

Reply:
That's a very clever configuration. After update from `0.7` to `1.0` ( in this PR or in the future)

```
julia_version: 1.0
julia_version: 1
```
always test the latest stable `1.x` release without manually change.

Reply:
I don't think it works on travis, sadly. Issue filed: https://github.com/JuliaCI/travis-build/issues/5

Reply:
Folks around me says windows CI is already supported on travis last year.

Reply:
Interesting. When I set this up that wasn't the case. I'd be fine with switching it all to travis.